rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java

```diff
@@ -450,6 +450,17 @@ public enum BridgeMethods {
             activations -> activations.isActive(RSKIP134),
             fixedPermission(true)
     ),
+    GET_ACTIVE_POWPEG_REDEEM_SCRIPT(
+            CallTransaction.Function.fromSignature(
+                    "getActivePowpegRedeemScript",
+                    new String[]{},
+                    new String[]{"bytes"}
+            ),
+            fixedCost(3_000L), // TODO: DEFINE THE CORRECT FIXED COST FOR THS METHOD
```

Comment:
Please, fix this before merge

Reply:
This will be completed in another task/PR. We already added the performance tests to get the data to define the cost(https://github.com/rsksmart/rskj/pull/1759).